src/commands.ts

```diff
@@ -513,50 +513,71 @@ export const run = async (functionName: string, cmd: { nondev: boolean }) => {
 
 /**
  * Deploys an Apps Script project.
- * @param version {string} The project version to deploy at.
- * @param description {string} The deployment's description.
+ * @param cmd.versionNumber {string} The project version to deploy at.
+ * @param cmd.desc {string} The deployment description.
+ * @param cmd.deploymentId  {string} The deployment ID to redeploy.
  */
-export const deploy = async (version: number, description = '') => {
+export const deploy = async (cmd :{
+  versionNumber: number,
+  desc: string,
+  deploymentId: string,
+}) => {
   await checkIfOnline();
   await loadAPICredentials();
   const { scriptId } = await getProjectSettings();
   if (!scriptId) return;
   spinner.setSpinnerTitle(LOG.DEPLOYMENT_START(scriptId)).start();
-  async function createDeployment(versionNumber: number) {
-    spinner.setSpinnerTitle(LOG.DEPLOYMENT_CREATE);
-    const deployments = await script.projects.deployments.create({
+
+  let versionNumber = cmd.versionNumber;
+  const description = cmd.desc || '';
+  const deploymentId = cmd.deploymentId;
+
```

Comment:
Replace the 3 lines here with:
```ts
let { versionNumber, description, deploymentId } = cmd;
```

Reply:
lint error : `Identifier 'desc' is never reassigned; use'const' instead of 'let'.`
I'll Write `const` and `let` separately.

```
let { versionNumber } = cmd;
const { desc='', deploymentId } = cmd;
```